pkg/query-service/tests/docker.go

```diff
@@ -0,0 +1,101 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	minio "github.com/minio/minio-go/v6"
+)
+
+const (
+	composeFile   = "./test-deploy/docker-compose.yaml"
```

Comment:
> composeFile   = "./test-deploy/docker-compose.yaml"

Using `runtime.GOARCH`, we can conditionally switch between that that and `docker-compose.arm.yaml`.

Reply:
Done